.github/workflows/merge-to-master.yml

```diff
@@ -218,11 +218,11 @@ jobs:
       matrix:
         include:
           - name: Windows Server 2019 (LTSC)
-            windowsImageTag: ltsc2019-amd64
+            windowsImageTag: ltsc2019
             imageTagSuffix: windows-ltsc-2019
             runsOn: windows-2019
           - name: Windows Server 2022 (LTSC)
-            windowsImageTag: ltsc2022-amd64
+            windowsImageTag: ltsc2022
```

Comment:
I guess the new images that we're using are multi-arch. Are we sure this is pulling the `amd64` equivalent? Is the GH runner always `amd64`? Should we configure this somewhere else?